stacks/apps/cms.yml

```diff
@@ -137,6 +138,65 @@ Resources:
       ManagedPolicyArns:
         - Fn::ImportValue: !Sub ${SecretsStackName}-SecretsInstanceDecryptPolicyArn
         - Fn::ImportValue: !Sub ${SecretsStackName}-SecretsInstanceAccessPolicyArn
+      Policies:
+        - PolicyDocument:
+            Statement:
+              - Action: sns:Publish
+                Effect: Allow
+                Resource: !Ref PorterJobExecutionSnsTopicArn
+            Version: "2012-10-17"
+          PolicyName: PorterSnsPublish
+        - PolicyDocument:
+            Statement:
+              - Action: sqs:*
+                Effect: Allow
+                Resource:
+                  - !GetAtt AudioCallbackQueue.Arn
+                  - !GetAtt ImageCallbackQueue.Arn
+                  - !GetAtt PodcastImportQueue.Arn
+                  - !GetAtt SearchIndexerQueue.Arn
+                  - !GetAtt DefaultJobQueue.Arn
+            Version: "2012-10-17"
+          PolicyName: JobQueues
+        - PolicyDocument:
+            Statement:
+              - Action: sqs:*
+                Effect: Allow
+                Resource: !Sub arn:aws:sqs:${AWS::Region}:${AWS::AccountId}:${AnnounceResourcePrefix}_${EnvironmentTypeAbbreviation}*
+              - Action: sns:*
+                Effect: Allow
+                Resource: !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:${AnnounceResourcePrefix}_${EnvironmentTypeAbbreviation}*
+            Version: "2012-10-17"
+          PolicyName: Announce
+        - PolicyDocument:
+            Statement:
+              - Action:
+                  - s3:GetObject
+                  - s3:GetObjectVersion
+                  - s3:ListBucket
+                  - s3:ListAllMyBuckets
+                Effect: Allow
+                Resource: "*"
+            Version: "2012-10-17"
+          PolicyName: S3ReadOnly
+        - PolicyDocument:
+            Statement:
+              - Action: s3:ListAllMyBuckets
+                Effect: Allow
+                Resource: "*"
+              - Action:
+                  - s3:AbortMultipartUpload
+                  - s3:DeleteObject
+                  - s3:Get*
+                  - s3:List*
+                  - s3:Put*
+                  - s3:RestoreObject
+                Effect: Allow
+                Resource: # Turn these into root stack paramters
+                  - !If [IsProduction, arn:aws:s3:::prx-feed/*, arn:aws:s3:::staging-prx-feed/*]
+                  - !If [IsProduction, arn:aws:s3:::prx-feed, arn:aws:s3:::staging-prx-feed]
```

Comment:
CMS needs write access to the feed bucket?  I thought it might read from other buckets ... but writes were always to the mediajoint bucket.

Reply:
This policy applied to both `prx-feed` and `prx-up` in the user, so that could have something to do with it. I copied the policies as they were, but would be glad for them to get tightened up if possible.